integrade/tests/api/v1/test_instances.py

```diff
@@ -70,20 +71,19 @@ def test_find_running_instances(drop_account_data, instances_to_terminate):
     list_images = client.get(urls.IMAGE, auth=auth)
     found_images = [image['ec2_ami_id'] for image in list_images['results']]
     source_image = \
-        aws_cfg['profiles'][aws_profile]['images']['rhel1']['image_id']
+        aws_profile['images']['rhel1']['image_id']
     assert source_image in found_images
 
-    timeout = 600
+    timeout = 30
```

Comment:
Did you meant 300? Not sure because down below this is sleeping for 60 seconds which is twice this timeout.